Games/types/Mafia/roles/cards/MeetYourMatch.js

```diff
@@ -19,6 +19,7 @@ module.exports = class MeetYourMatch extends Card {
       "Lovebird B": {
         states: ["Night"],
         flags: ["voting"],
+        targets: { include: ["alive"], exclude: ["self", isLovebirdA] },
```

Comment:
targets are generated at the start of the phase

it's better to change this to multimeeting like justice